faker/providers/lorem/__init__.py

```diff
@@ -1,67 +1,108 @@
 localized = True
-default_locale = 'la'
+#default_locale is 'en_US' in the previous State of this application
+default_locale = 'la' 
+#external provider
+external_provider = ''
 
 from .. import BaseProvider
 
 
 class Provider(BaseProvider):
+    '''
+    factory.create will enter this class and will and will list all the
+    methods then will go to 'la.__init__'
+
+    Why does it goes to 'la' as soon as it finish to read this class
+    '''
+    #import pytest; pytest.set_trace()
     word_connector = ' '
     sentence_punctuation = '.'
     @classmethod
-    def word(cls):
+    def word(cls, ext_word_list=None):
         """
         Generate a random word
         :example 'lorem'
+
+        Keyword arguments:
+        'ext_word_list' --- a list of word you would like to have 
+                 instead of 'Lorem ipsum'
         """
+        if ext_word_list:
+            return cls.random_element(ext_word_list)
         return cls.random_element(cls.word_list)
 
     @classmethod
-    def words(cls, nb=3):
+    def words(cls, nb=3, ext_word_list=None):
         """
         Generate an array of random words
         :example array('Lorem', 'ipsum', 'dolor')
-        :param nb how many words to return
+
+        Keyword arguments:
+        nb --- how many words to return
+        'ext_word_list' --- a list of word you would like to have 
+                 instead of 'Lorem ipsum'
         """
+        if ext_word_list:
+            return [cls.word(ext_word_list) for _ in range(0, nb)]
```

Comment:
you don't need this `if` clause, as `.word()` takes care of it already.

Simply modify the line below to call `[cls.word(ext_word_list) for _ in range(0, nb)]`